### PR TITLE
 Changes in docs/source/tasks files

### PR DESCRIPTION
**Pull request by shubhamshukla25**
Changes made to the folder docs/source/tasks

### CODE REVIEW

File: docs/source/tasks/intermediateCA.rst [hunk 1]
```diff
@@ -5,7 +5,7 @@
 Creating an Intermediate Certificate Authority
 =================================================
 
-IBM Blockchain Platform allows for the optional use of Intermediate CAs which can protect a root CA and provide operational flexibility. The IBM Blockchain Platform documentation describes the business rerquirement and the manual steps in detail.
+IBM Support for Hyperledger Fabric allows for the optional use of Intermediate CAs which can protect a root CA and provide operational flexibility. The IBM Support for Hyperledger Fabric documentation describes the business rerquirement and the manual steps in detail.
```
Comment (modemn) [inline docs/source/tasks/intermediateCA.rst:8]: Similar thing here, need to generalise the wording since this applies to the Fabric OSS as well

Comment (modemn) [reply docs/source/tasks/intermediateCA.rst:8]: Also might want to get rid of the reference to the documentation since this is a Hyperledger Fabric repo

**Review by modemn — CHANGES_REQUESTED** [review]
All the links for the playbooks also still point to the IBM-Blockchain/ansible-collection repo which should be changed to the new collection repo.

File: docs/source/tasks/intermediateCA.rst [hunk 2]
```diff
@@ -21,7 +21,7 @@ Before you start
 
 This task guide assumes that you have installed Ansible and the Hyperledger Fabric Ansible Collection, and are familiar with how to use these technologies.
 
-This task guide assumes that you have access to the IBM Blockchain Platform documentation and the sections regarding Intermediate CAs.
+This task guide assumes that you have access to the IBM Support for Hyperledger Fabric documentation and the sections regarding Intermediate CAs.
```
[the remark this answers is not in the document]
Comment (modemn) [reply docs/source/tasks/intermediateCA.rst:24]: I think we can just get rid of this line.

File: docs/source/tasks/addadmincert.rst [hunk 3]
```diff
@@ -145,7 +135,7 @@ Depending on the role of the organization being updated, perform the appropriate
 
 **If the organization is an administrator of the ordering service**
 
-  Review the example playbook `05-update-channels-admin.yml <https://github.com/IBM-Blockchain/ansible-collection/blob/main/examples/add-admin-cert/05-update-channels-admin.yml>`_, then run it as follows:
+  Review the example playbook `05-update-channels-admin.yml <https://github.com/IBM-Blockchain/fabric-ansible-collection/blob/main/examples/add-admin-cert/05-update-channels-admin.yml>`_, then run it as follows:
```
[the remark this answers is not in the document]
Comment (modemn) [reply docs/source/tasks/addadmincert.rst:138]:
```suggestion
  Review the example playbook `05-update-channels-admin.yml <https://github.com/hyperledger-labs/fabric-ansible-collection/blob/main/examples/add-admin-cert/05-update-channels-admin.yml>`_, then run it as follows:
```

**Review by modemn — CHANGES_REQUESTED** [review]

File: docs/source/tasks/intermediateCA.rst [hunk 4]
```diff
@@ -5,7 +5,7 @@
 Creating an Intermediate Certificate Authority
 =================================================
 
-IBM Blockchain Platform allows for the optional use of Intermediate CAs which can protect a root CA and provide operational flexibility. The IBM Blockchain Platform documentation describes the business rerquirement and the manual steps in detail.
+Hyperledger Fabric allows for the optional use of Intermediate CAs which can protect a root CA and provide operational flexibility. The  Hyperledger Fabric CA documentation describes the business rerquirement and the manual steps in detail.
```
[the remark this answers is not in the document]
Comment (shubhamshukla25) [reply docs/source/tasks/intermediateCA.rst]: Changes committed as per the review comments

File: docs/source/tasks/update-channel-parameters.rst [hunk 5]
```diff
@@ -5,7 +5,7 @@
 Updating channel parameters
 ===========================
 
-Hyperledger Fabric allows each channel to be configured with specific BatchSize parameters and a BatchTimeout for blocks.  This playbook example shows how an Architect or Chaincode Developer could to set these values for a particular application channel on IBM Blockchain Platform.
+Hyperledger Fabric allows each channel to be configured with specific BatchSize parameters and a BatchTimeout for blocks.  This playbook example shows how an Architect or Chaincode Developer could to set these values for a particular application channel on IBM Support for Hyperledger Fabric software.
```
[the remark this answers is not in the document]
Comment (shubhamshukla25) [reply docs/source/tasks/update-channel-parameters.rst]: Changes committed as per the review comments

**Review by modemn — APPROVED** [review]

**Pull request merged**